### PR TITLE
[docs] Doc about expand entities in investigation graph (#issue/1865)

### PR DESCRIPTION
**Pull request by lndrtrbn**
Related to https://github.com/OpenCTI-Platform/opencti/issues/1865

### CODE REVIEW

File: docs/usage/pivoting.md [hunk 1]
```diff
@@ -13,17 +13,28 @@ In your workspace, you can add entities that you want to investigate, visualize
 
 ![Investigation workspace](assets/investigation-workspace.png)
 
+On some nodes, you can see next to them a bullet with a number inside. It is a visual indication that shows you how many entities have relationship with this one.
```
Comment (Archidoit) [inline docs/usage/pivoting.md:16]: have relationshipS

Comment (Archidoit) [inline docs/usage/pivoting.md:16]: have relationshipS

Comment (RomuDeuxfois) [inline docs/usage/pivoting.md:16]: Why some ? Is not for everyone ?

File: docs/usage/pivoting.md [hunk 2]
```diff
@@ -13,17 +13,28 @@ In your workspace, you can add entities that you want to investigate, visualize
...
 ![Investigation bottom right menu](assets/investigation-bottom-right-menu.png)
 
-Once added, you can select the entity, and see its details in the right. 
-In this bottom right menu, right next to "Add en entity", you can expand the selected entity and select the number of linked entities you want to see in your investigation.
+Once added, you can select the entity, and see its details in the panel that appears on the right of the screen.
+
+
+In the same menu as above, right next to "Add en entity", you can expand the selected entity. Clicking on the menu icon open a new window where you can choose which type of entities and relationships you want to expand.
+
+For each type of entity or relationship, it is displayed in parentheses the number of elements that will be added into the investigation graph. This time there is no `~` symbol as the number is no more an approximation at this step.
```
Comment (Archidoit) [inline docs/usage/pivoting.md:32]: the number of elements that will be added into the investigation graph is displayed in parentheses

**Review by guillaumejparis — APPROVED** [review]

File: docs/usage/pivoting.md (hunk 1 above)
Comment (lndrtrbn) [reply docs/usage/pivoting.md:16]: There is no bullet displayed when nothin to expand, but maybe I can just remove "On some nodes" if it is confusing

File: docs/usage/pivoting.md (hunk 2 above)
Comment (Archidoit) [inline docs/usage/pivoting.md:32]: 'this time' and 'this step' are a bit repetitive in the same sentence

File: docs/usage/pivoting.md (hunk 1 above)
Comment (lndrtrbn) [reply docs/usage/pivoting.md]: Removed

File: docs/usage/pivoting.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lndrtrbn) [reply docs/usage/pivoting.md]: Changes done in the commit just pushed

**Review by labo-flg — APPROVED** [review]

File: docs/usage/pivoting.md [hunk 3]
```diff
@@ -13,17 +13,28 @@ In your workspace, you can add entities that you want to investigate, visualize
 
 ![Investigation workspace](assets/investigation-workspace.png)
 
+You can see next to them a bullet with a number inside. It is a visual indication that shows you how many entities have relationships with this one.
```
Comment (Jipegien) [inline docs/usage/pivoting.md:16]:
```suggestion
You can see next to them a bullet with a number inside. It is a visual indication showing you how many entities are linked to this one and not displayed in the graph yet.
```

File: docs/usage/pivoting.md [hunk 4]
```diff
@@ -13,17 +13,28 @@ In your workspace, you can add entities that you want to investigate, visualize
...
+Note that this number is an approximation of the number of entities. That's why there is a `~` next to the number.
+
+No bullet displayed means: nothing to expand from this node.
```
Comment (Jipegien) [inline docs/usage/pivoting.md:19]:
```suggestion
No bullet displayed means there is nothing to expand from this node.
```

File: docs/usage/pivoting.md [hunk 5]
```diff
@@ -13,17 +13,28 @@ In your workspace, you can add entities that you want to investigate, visualize
...
 ![Investigation bottom right menu](assets/investigation-bottom-right-menu.png)
 
-Once added, you can select the entity, and see its details in the right. 
-In this bottom right menu, right next to "Add en entity", you can expand the selected entity and select the number of linked entities you want to see in your investigation.
+Once added, you can select the entity, and see its details in the panel that appears on the right of the screen.
+
+
+In the same menu as above, right next to "Add en entity", you can expand the selected entity. Clicking on the menu icon open a new window where you can choose which type of entities and relationships you want to expand.
+
+For each type of entity or relationship, the number of elements that will be added into the investigation graph is displayed in parentheses. This time there is no `~` symbol as the number is no more an approximation.
```
Comment (Jipegien) [inline docs/usage/pivoting.md:32]:
```suggestion
For each type of entity or relationship, the number of elements that will be added into the investigation graph is displayed in parentheses. This time there is no `~` symbol as the number is exact.
```

**Pull request merged**